### PR TITLE
Added documentation for EBB LED blink patterns

### PR DESCRIPTION
**Pull request by EmbeddedMan**

### CODE REVIEW

File: docs/ebb.html [hunk 1]
```diff
@@ -2888,6 +2888,75 @@ <h2><a name="faq"></a>Frequently Asked Questions</h2>
...
+              </tr>
+              <tr>
+                <td>Short Long Blink</td>
+                <td>Fully enumerated and communicating with USB host</td>
+                <td>On</td>
+                <td>365 ms on, 365 ms off, 1.25s on, 365 ms off</td>
+              </tr>
+            </tbody>
+          </table>
+          The Fast Blink pattern is almost never seen in practice, since the USB host normally enumerates the EBB immediately upon connection with a USB cable. However, if proper drivers are not installed on the host or if there is some other problem with the USB enumeration process the Fast Blink pattern can be observed and used as a debugging aid.
```
Comment (oskay) [inline docs/ebb.html:2957]: There's something a little off on the formatting of the last section (" The Fast Blink pattern..."); the indentation is lower than it should be, and does not match the text above.

File: docs/ebb.html [hunk 2]
```diff
@@ -2888,6 +2888,75 @@ <h2><a name="faq"></a>Frequently Asked Questions</h2>
...
+              <tr style="font-weight:bold;">
+                <td>Pattern</td>
+                <td>Description</td>
+                <td>USR/Red</td>
+                <td>USB/Green</td>
+              </tr>
+              <tr>
+                <td>Fast Blink</td>
+                <td>No connection to USB host</td>
+                <td>On</td>
```
Comment (oskay) [inline docs/ebb.html:2940]: I'm not seeing the red LED/USR LED on in these cases.

Comment (EmbeddedMan) [reply docs/ebb.html:2940]: See video I just sent via Discord.

File: docs/ebb.html (hunk 1 above)
Comment (EmbeddedMan) [reply docs/ebb.html:2959]: File updated with fix. Should be good now.

File: docs/ebb.html (hunk 2 above)
Comment (EmbeddedMan) [reply docs/ebb.html]: OK, PR updated with red LED shown as off all the time.

**Review by oskay — APPROVED** [review]

**Comment by EmbeddedMan** [comment]
See issue #194

**Pull request merged**